go-runtime/compile/schema_test.go

```diff
@@ -50,24 +49,44 @@ func TestParseDirectives(t *testing.T) {
 	}{
 		{name: "Module", input: "ftl:module foo", expected: &directiveModule{Name: "foo"}},
 		{name: "Verb", input: "ftl:verb", expected: &directiveVerb{Verb: true}},
-		{name: "IngressImplicitFTL", input: `ftl:ingress GET /foo`, expected: &directiveIngress{
-			Type: &directiveIngressHTTP{
+		{name: "Ingress", input: `ftl:ingress GET /foo`, expected: &directiveIngress{
+			MetadataIngress: schema.MetadataIngress{
+				Pos: schema.Position{
```

Comment:
Instead of including the positions, which get very difficult to maintain, add another `assert.exclude[schema.Position]()` option to the comparison below.